bot.py

```diff
@@ -264,82 +261,93 @@ async def add_summoner(ctx, *, message):
         async with ctx.typing():
             await asyncio.sleep(1)
 
-        # create a directory containing json file to store data for added summoners
-        if not os.path.exists(json_path):
-            os.makedirs(data_folder_path, exist_ok=True)
-            with open(json_path, "w"):
-                pass
-
         # converting the message into list of summoners
-        player_list = message.split(",")
```

Comment:
@HelloFive @jung1411 @jjaykim 
For future reference. Spend sometime thinking about variable names. 

It was extremely difficult to follow the code base
each names should clearly represent what it's containing.
Of course we each have personal preference but I think it can be better
eg
`player_list` -> `user_input_names`

Reply:
Okay.!